CHANGES.md

```diff
@@ -1,6 +1,22 @@
 # git master
 
-* cmake 2.8.9 or later is now required
+# 2015.11
+
+* cmake 2.8.9 or later is now required; reduced cmake 3 warnings
+* cmake 3 is required for installing projects correctly (@tribal-tec
+  correct?)
```

Comment:
...for finding a project with find_package() from outside the build tree/from the install tree.